.github/workflows/master-e2e.yaml

```diff
@@ -394,6 +399,7 @@ jobs:
           echo "Rancher Manager Version: ${RM_VERSION}" >> $GITHUB_STEP_SUMMARY
           echo "Elemental Operator Upgrade: ${UPGRADE_OPERATOR_VALUE}" >> $GITHUB_STEP_SUMMARY
           echo "Elemental Operator Version: ${OPERATOR_VERSION}" >> $GITHUB_STEP_SUMMARY
+          echo "Elemental UI Extension Version (if applicable): ${{ inputs.elemental_ui_version || 'N/A'}}" >> $GITHUB_STEP_SUMMARY
```

Comment:
A space is missing, more `${{ inputs.elemental_ui_version || 'N/A' }}"` (but I'm not sure if this has a real impact in fact).